code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -875,6 +875,22 @@
 	inverse_chem_val = 0.45
 	impure_chem = /datum/reagent/impurity/mannitol
 
+/datum/reagent/medicine/mannitol/expose_obj(obj/exposed_obj, reac_volume)
+	. = ..()
+	var/obj/item/organ/brain/exposed_brain = exposed_obj
+	if(!exposed_brain)
+		return
+
+	if(exposed_brain.brainmob?.health <= HEALTH_THRESHOLD_DEAD) //if the brain is fucked anyway, do nothing
+		to_chat(usr, "<span class='warning'>[exposed_brain] is far too damaged, there's nothing else we can do for it!</span>")
```

Comment:
There might not be a `usr` to message, I wouldn't produce any message in this case.